### PR TITLE
bsp/nrf52840pdk: replace bitbang by native uart for UART1

### PR DESCRIPTION
**Pull request by sdalu**
Fix for: https://github.com/apache/mynewt-core/issues/1043

### CODE REVIEW

File: hw/bsp/nrf52840pdk/src/hal_bsp.c [hunk 1]
```diff
@@ -315,9 +313,9 @@ hal_bsp_init(void)
     assert(rc == 0);
 #endif
 
-#if MYNEWT_VAL(UART_1)
-    rc = os_dev_create((struct os_dev *) &os_bsp_bitbang_uart1, "uart1",
-      OS_DEV_INIT_PRIMARY, 0, uart_bitbang_init, (void *)&os_bsp_uart1_cfg);
+#if MYNEWT_VAL(UART_0)
```
Comment (andrzej-kaczmarek) [inline hw/bsp/nrf52840pdk/src/hal_bsp.c]: should be 'UART_1'

File: hw/bsp/nrf52840pdk/syscfg.yml [hunk 2]
```diff
@@ -40,14 +40,20 @@ syscfg.defs:
...
+        description: 'Whether to enable UART1'
         value:  0
     UART_1_PIN_TX:
         description: 'TX pin for UART1'
         value:  -1
     UART_1_PIN_RX:
         description: 'RX pin for UART1'
         value:  -1
+    UART_1_PIN_RTS:
+        description: 'RTS pin for UART0'
```
Comment (andrzej-kaczmarek) [inline hw/bsp/nrf52840pdk/syscfg.yml]: `UART1`

File: hw/bsp/nrf52840pdk/syscfg.yml [hunk 3]
```diff
@@ -40,14 +40,20 @@ syscfg.defs:
...
+        value:  -1
+    UART_1_PIN_CTS:
+        description: 'CTS pin for UART0'
```
Comment (andrzej-kaczmarek) [inline hw/bsp/nrf52840pdk/syscfg.yml]: `UART1`

**Comment by sdalu** [comment]
oops... fixed

**Pull request merged**